libbeat/docs/shared-passing-credentials-setup.asciidoc

```diff
@@ -0,0 +1,48 @@
+//REVIEWERS: This content underscores a problem in our docs where we recommend using sudo, but don't show sudo in all the examples (because some of the examples come from shared content like this topic). It's confusing to users when we show some commands with sudo and some without. I can conditionally code the content to make sudo part of the command only when it's required, but I want to make sure we have a full list of the Beats that require sudo. Can we identify a list of Beats where we expect users to always run as sudo? There's also the problem of providing platform-agnostic command examples.
```

Comment:
Most of the Beats require to sudo commands. Exception is Filebeat and Heartbeat (only if icmp is not configured).